specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2023-07-01/redisenterprise.json

```diff
@@ -0,0 +1,1878 @@
+            "name": "AccessKeyType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "keyType"
+      ],
+      "description": "Specifies which access keys to reset to a new random value."
+    },
+    "ImportClusterParameters": {
+      "title": "Import RDB files into a target database",
+      "type": "object",
+      "properties": {
+        "sasUris": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "SAS URIs for the target blobs to import from",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUris"
+      ],
+      "description": "Parameters for a Redis Enterprise import operation."
+    },
+    "ExportClusterParameters": {
+      "title": "Export an RDB file into a target database",
+      "type": "object",
+      "properties": {
+        "sasUri": {
+          "type": "string",
+          "description": "SAS URI for the target directory to export to",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUri"
+      ],
+      "description": "Parameters for a Redis Enterprise export operation."
+    },
+    "OperationStatus": {
+      "description": "The status of a long-running operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The operation's unique id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation's name."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the operation."
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the operation."
+        },
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse",
+          "description": "Error response describing why the operation failed."
+        }
+      }
+    },
+    "ForceUnlinkParameters": {
+      "title": "Forcibly unlink another database from this database.",
+      "type": "object",
+      "properties": {
+        "ids": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The resource IDs of the database resources to be unlinked."
+        }
+      },
+      "required": [
+        "ids"
+      ],
+      "description": "Parameters for a Redis Enterprise Active Geo Replication Force Unlink operation."
+    },
+    "FlushParameters": {
+      "title": "Flush all the keys from the current and all its linked databases.",
+      "type": "object",
+      "properties": {
+        "ids": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
Is it an azure resource ID? If so, please add "format": "arm-id" and "x-ms-arm-id-details".

https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details

Reply:
yes. It should be list of databases!